YOUR_ADMIN/edit_orders.php

```diff
@@ -56,24 +56,9 @@
 require DIR_WS_CLASSES . 'editOrders.php';
 $eo = new editOrders($oID);
 
-$orders_statuses = [];
-$orders_status_array = [];
-$order_by_field = ($sniffer->field_exists(TABLE_ORDERS_STATUS, 'sort_order')) ? 'sort_order' : 'orders_status_id';
-$orders_status_query = $db->Execute(
-    "SELECT orders_status_id, orders_status_name
-       FROM " . TABLE_ORDERS_STATUS . "
-      WHERE language_id = " . (int)$_SESSION['languages_id'] . "
-  ORDER BY $order_by_field ASC"
-);
-foreach ($orders_status_query as $orders_status) {
-    $status_id = $orders_status['orders_status_id'];
-    $status_name = $orders_status['orders_status_name'];
-    $orders_statuses[] = [
-        'id' => $status_id,
-        'text' => "$status_name [$status_id]"
-    ];
-    $orders_status_array[$status_id] = $status_name;
-}
+$ordersStatus = zen_getOrdersStatuses();
```

Comment:
For which Zen Cart release was this function introduced?

Reply:
i see it in the v2.0.0 tag:

https://github.com/zencart/zencart/blob/7e9875b2ec20a124b7dc703ee32311fb6cb238a5/admin/includes/functions/general.php#L714-L733

Reply:
I'll be incorporating this into EO 5.0.0, noting that the syntax for usability needs to be the following for the rest of the EO code to make use of that update:
```php
['orders_statuses' => $orders_statuses, 'orders_status_array' => $orders_status_array] = zen_getOrdersStatuses();
```

Reply:
do it however you want; but in my testing the **3 lines** of code that i added allow the rest of EO to make use of it. 

you want to do it in 1 line, that's fine with me; it is your repo.  but this PR is 3 lines of code; and if you want to note it will not work with the rest of EO, then i would like to see some proof of that.

Reply:
Logs similar to the following, using the 5.0.0 branch:
```
[15-Sep-2024 20:16:59 America/New_York] Request URI: /zc210/admin210/index.php?cmd=edit_orders&page=1&action=edit&oID=18, IP address: 127.0.0.1, Language id 1
#0 C:\xampp\htdocs\zc210\zc_plugins\EditOrders\v5.0.0\admin\includes\modules\edit_orders\eo_edit_action_osh_table_display.php(156): zen_debug_error_handler()
#1 C:\xampp\htdocs\zc210\zc_plugins\EditOrders\v5.0.0\admin\includes\modules\edit_orders\eo_edit_action_display.php(497): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\zc210\zc_plugins\EditOrders\v5.0.0\admin\edit_orders.php(167): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\zc210\admin210\index.php(28): require('C:\\xampp\\htdocs...')
--> PHP Warning: Undefined variable $orders_status_array in C:\xampp\htdocs\zc210\zc_plugins\EditOrders\v5.0.0\admin\includes\modules\edit_orders\eo_edit_action_osh_table_display.php on line 156.
```

Reply:
if you want to test a PR on your local development environment, i use something similar to:

```
git pull upstream pull/250/head:new-branch-name
```

if you want to manually test it, i would suggest you copy all 3 lines of the code which are as follows:

```
$ordersStatus = zen_getOrdersStatuses();
$orders_status_array = $ordersStatus['orders_status_array'];
$orders_statuses = $ordersStatus['orders_statuses'];
```

i am not sure how you would have an undefined `$orders_status_array` as it looks to be defined on line 2 of this PR.

am i missing something?